Source/Core/VideoCommon/BPMemory.h

```diff
@@ -1050,24 +1050,62 @@ struct fmt::formatter<LPSize>
   }
 };
 
-union X12Y12
+union ScissorPos
+{
+  // The top bit is ignored, and not included in the mask used by SDK functions
+  // (though libogc includes it for the bottom coordinate (only) for some reason)
+  // The SDK also adds 342 to this value.
+  BitField<0, 11, u32> y;
+  BitField<0, 12, u32> y_full;
```

Comment:
The `_full` members could use a short comment explaining what distinguishes them from the non-`_full` ones, if that's information we know.

Reply:
There isn't any information other than the top bit being ignored - `y` is `y_full` with that bit ignored.  But the comment doesn't currently make that clear.